syncdb.drush.inc

```diff
@@ -210,7 +209,7 @@ function drush_syncdb_importdb() {
   }
   else {
     // Manual import. Still faster than sql-sync.
-    drush_log(dt('Importing tables manually. This is still faster than sql-sync but for ludicrous speed install http://www.gnu.org/software/parallel', 'info'), 'status');
+    drush_log(dt('Importing tables manually. This is still faster than sql-sync but for ludicrous speed install http://www.gnu.org/software/parallel', 'info'));
```

Comment:
What's the motive for this change? I know I appreciated this message the first time I ran the command.